web/src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import Header from "../../components/common/Header";
+import Footer from "../../components/common/Footer";
+
+class NotFoundPage extends Component {
+  render() {
+    return (
+      <div>
+        <header>
+          <Header />
+        </header>
+
+        <div className="flexContainer wrapper">
```

Comment:
Why are we using camelCase for the class names?

Reply:
Yeah, we should clean that up. Just created #52 for that!

Reply:
Created PR!